sparql11/data-sparql11/functions/encode01-non-bmp.srx

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<sparql xmlns="http://www.w3.org/2005/sparql-results#">
+<head>
+	<variable name="s"/>
```

Comment:
Note: tabs don't display nicely in github interface.

Reply:
Will fix in all srx files.